### PR TITLE
WIP unit test in parallel for CI gates 

### PR DESCRIPTION
**Pull request by benbp**

### CODE REVIEW

File: eng/pipelines/templates/jobs/archetype-sdk-client.yml [hunk 1]
```diff
@@ -149,7 +152,7 @@ jobs:
       matrix:
         Linux:
           Pool: azsdk-pool-mms-ubuntu-1804-general
-          OSVmImage: MMSUbunut18.04
```
Comment (mitchdenny) [inline eng/pipelines/templates/jobs/archetype-sdk-client.yml:152]: Whooops!

**Comment by mitchdenny** [comment]
Is this just an experiment?

**Comment by benbp** [comment]
Yeah sorry for the notification spam @mitchdenny

File: eng/AzSdk.test.reference.targets [hunk 2]
```diff
@@ -12,7 +12,4 @@
     <!-- This is needed for discovering tests in test explorer -->
     <PackageReference Include="System.Runtime.InteropServices" />
   </ItemGroup>
-  <ItemGroup>
```
Comment (weshaggard) [inline eng/AzSdk.test.reference.targets:15]: I don't think these are doing anything for our projects I think it is only for track 1.

**Pull request closed without merging**